server/utils/breadcrumbUtil.js

```diff
@@ -25,7 +32,7 @@ function createAboutCourseItem(language, courseCode) {
 }
 
 function createBreadcrumbs(language, courseCode) {
-  const items = [baseItems.university[language], baseItems.student[language], baseItems.directory[language]]
+  const items = [baseItems(language).student, baseItems(language).studies, baseItems(language).directory]
```

Comment:
Nice that you converted baseItems into a function. But since it's now a function i think its better to just call it once and assign to a variable on the line before. And rename the function to `createBaseItems`.

Like this:
```
const baseItems = createBaseItems(language)
const items = [baseItems.student, baseItems.studies, baseItems.directory]
```

Reply:
Now I made the required adjustments and also I updated the url for studies.

Reply:
Ah good that you find the URL for studies as well, I missed that!